sdk/communication/Azure.Communication.CallingServer/src/CallingServerClientOptions.cs

```diff
@@ -39,7 +39,7 @@ public enum ServiceVersion
             /// The Beta of the CallingServer service.
             /// </summary>
 #pragma warning disable CA1707 // Identifiers should not contain underscores
-            V2021_04_15_Preview1 = 0
+            V2021_06_15_Preview = 1
```

Comment:
why is the enum starting with 1 but the 2021_04_15 version is removed?

Reply:
This number started from 1 was per Azure boarder team suggest, they seems reserve 0 for other usage. CC @navali-msft 

"2021_04_15" was a "private review" version. There are plenty of major schema changes on the swagger and since we are not public preview yet, we decide to have "V2021_06_15_Preview" as our first public version.